recipes/java-access-bridge-wrapper/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "java-access-bridge-wrapper" %}
+{% set version = "0.14.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/java_access_bridge_wrapper-{{ version }}.tar.gz
+  sha256: 8ee085b6257b23037e84c0572821e636402b14450ec7d396b0b04ab85ce40476
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - java_tree_reader = 'JABWrapper.context_tree_reader:main'
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<4.0
+    - poetry >=0.12
+    - poetry-core
+    - pip
+  run:
+    - python >=3.6.2,<4.0.0
+    - pywin32-on-windows
+    - pywin32 >=300,<304  # [win]
+    - __win    # [win]
```

Comment:
```suggestion
    - pywin32 >=300,<304
    - __win
```

Reply:
@rxm7706 you will need to add,

```yaml
noarch_platforms:
  - win_64
  ```
  
  to the `conda-forge.yaml` in the feedstock after the merge here and re-render.

Reply:
Thanks @ocefpaf will add 

> @rxm7706 you will need to add,
> 
> ```yaml
> noarch_platforms:
>   - win_64
> ```
> 
> to the `conda-forge.yaml` in the feedstock after the merge here and re-render.